Sources/TuistPlugin/PluginService.swift

```diff
@@ -16,21 +17,24 @@ public protocol PluginServicing {
 /// A default implementation of `PluginServicing` which loads `Plugins` using the `Config` manifest.
 public final class PluginService: PluginServicing {
     private let manifestLoader: ManifestLoading
+    private let templateDirectoryLocator: TemplatesDirectoryLocating
     private let fileHandler: FileHandling
     private let gitHandler: GitHandling
 
     /// Creates a `PluginService`.
     /// - Parameters:
-    ///   - manifestLoader: A manifest loader for loading plugins.
-    ///   - configLoader: A configuration loader
+    ///   - manifestLoader: A manifest loader for loading plugin manifests.
+    ///   - templateDirectoryLocator: Locator for finding templates for plugins.
     ///   - fileHandler: A file handler for creating plugin directories/related files.
     ///   - gitHandler: A git handler for cloning and interacting with remote plugins.
     public init(
         manifestLoader: ManifestLoading = ManifestLoader(),
+        templateDirectoryLocator: TemplatesDirectoryLocating = TemplatesDirectoryLocator(),
```

Comment:
Nit: I'd rename `templateDirectoryLocator` to `templatesDirectoryLocator` to be consistent with the type name